sdk/include/hipdnn_sdk/test_utilities/FlatbufferDatatypeMapping.hpp

```diff
@@ -0,0 +1,100 @@
+// Copyright © Advanced Micro Devices, Inc., or its affiliates.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <hipdnn_sdk/data_objects/data_types_generated.h>
+#include <hipdnn_sdk/utilities/UtilsBfp16.hpp>
+#include <hipdnn_sdk/utilities/UtilsFp16.hpp>
+
+namespace
+{
+template <typename T>
```

Comment:
Please explain the purpose of this struct with a comment. It's not clear to me what this is useful for, it looks like an extra layer of indirection. Literally "TypeWrapper<T> has an alias "type" that is an alias for T." Why do this?